site/_shortcodes/LanguageList.js

```diff
@@ -36,18 +35,10 @@ function LanguageList(url, site, collections, locale = 'en') {
   const links = hreflangs
     .filter(([, code]) => code !== locale)
     .map(([urlPath, code]) => {
-      return `<a translate="no" lang="${code}"
-        href="${path.join(site.url, urlPath)}">
-        ${languageNames[code]}</a>`;
+      return `<a translate="no" lang="${code}" href="${urlPath}">${languageNames[code]}</a>`;
```

Comment:
can't comment on unedited code. In this case, the `const cleanUrl = stripLocale(url);` step is not necessary. I added the `stripLocale` util in last PR. Can remove that as well.

Reply:
It is redundant because `getLocalizedPaths` does it too, but I've left it in with a comment. I think it makes the code clearer.